TVOAppliance/hf_upgrade.sh

```diff
@@ -95,6 +95,25 @@ function reconfigure_s3_service_path()
 	sed  -i "s~$src_string~$dest_string~g" $file_name
 
 }
+
+#function to keep only tls1_2 access
+function limit_access_to_tls1_2()
+{
+	config_file="/etc/nginx/nginx.conf"
+	config_line="ssl_protocols TLSv1.2;"
+
+	# Check if the line already exists in the file
+	if grep -q "$config_line" "$config_file"; then
```

Comment:
add http check in the grep command 
grep -A4 http nginx.conf | grep -q "ssl_protocols TLSv1.2" 
add regex for HTTP<space> {